src/minimega/plumber.go

```diff
@@ -188,7 +188,7 @@ func cliPipeBroadcast(ns *Namespace, c *minicli.Command, resp *minicli.Response)
 		}
 	} else {
 		// get info on all named pipes
-		resp.Header = []string{"name", "mode", "readers", "writers", "via", "last message"}
+		resp.Header = []string{"name", "mode", "readers", "writers", "via", "previous", "count"}
```

Comment:
Let's change the order so that `count` comes after `writers`.

Reply:
Will do.